dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -207,6 +220,7 @@ def test_stream_profilers(self, *mocks):
         self.assertEqual(0, profiler.row_is_null_count)
         self.assertEqual(3, profiler.total_samples)
         self.assertEqual(2, len(profiler.hashed_row_dict))
+        self.assertIsNone(profiler.correlation_matrix)
```

Comment:
considering the update is mocked out, not sure theres a reason to validate the `correlation_matrix`

Reply:
agreed, removed it. Originally I kept it without the mock